src/locales/en/messages.json

```diff
@@ -4186,5 +4186,39 @@
   },
   "updateMasterPasswordWarning": {
     "message": "Your Master Password was recently changed by an administrator in your organization. In order to access the vault, you must update your Master Password now. Proceeding will log you out of your current session, requiring you to log back in. Active sessions on other devices may continue to remain active for up to one hour."
+  },
+  "maximumVaultTimeout": {
+    "message": "Vault Timeout"  
+  },
+  "maximumVaultTimeoutDesc": {
+    "message": "Configure an automatic vault timeout for all users based on a maximum duration."  
```

Comment:
Isn't something like "Configure a maximum automatic vault timeout for all users." closer?

I don't believe we're setting the vault timeout TO the provided value for all user, just requiring timeout to be at most the given value.

Reply:
I agree it's weirdly phrased, it's taken from the Asana ticket, but I'll check if we can change it.